web/packages/teleterm/src/ui/ConnectMyComputer/CompatibilityPromise.tsx

```diff
@@ -49,7 +49,7 @@ export function checkAgentCompatibility(
     : 'incompatible';
 }
 
-export function CompatibilityError(): JSX.Element {
+export function CompatibilityError(props: { showAlert: boolean }): JSX.Element {
```

Comment:
`hideAlert?: boolean` maybe? Usually we want to show the alert, so perhaps it is better to crate a special case for hiding it?

Reply:
Yeah, I think I'm going to do `hideAlert` with a default to false.